src/utils/format.js

```diff
@@ -17,9 +17,16 @@
 
 import { format } from 'd3-format';
 
+export function displayDecimal(val, places) {
```

Comment:
Not a huge fan of this function name/overloading. Confusing to use a function called `displayDecimal` for displaying an integer when no `places` provided. Can we at least think of a different name?